Bhagyashri/Project-WHO/WHO website/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE HTML>
+<html>
+ <head>
+   <title> WHO World Health Oraganization </title>
+   <link rel="shortcut icon" href="/image/favicon.ico" type="image/x-icon">
+   <link rel="stylesheet" type="text/css" href="./css/first.css"/>
+ </head>
+ <body>
+ <section class="sec-first">
```

Comment:
avoid using first second in class name frequently use the proper name of the section it includes.